### PR TITLE
Update to support agent version 1.13.0

### PR DESCRIPTION
**Pull request by nmeyerhans**

### CODE REVIEW

**Comment by aaithal** [comment]
lgtm :shipit:

File: ecs-init/VERSION [hunk 1]
```diff
@@ -0,0 +1 @@
+1.12.2
```
Comment (samuelkarp) [inline ecs-init/VERSION]: Can you update this to `1.13.0` and run the `update-version.sh` script to generate a new version file?

Comment (nmeyerhans) [inline ecs-init/VERSION]: done. please refresh this PR.

File: ecs-init/version/version.go [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+package version
+
+// Version is the version of the ecs-init
+const Version = "1.13.0"
+
+// GitDirty indicates the cleanliness of the git repo when this ecs-init was built
+const GitDirty = true
+
+// GitShortHash is the short hash of this ecs-init build
+const GitShortHash = "c153a82"
```
Comment (samuelkarp) [inline ecs-init/version/version.go]: I think we missed copying this over in @richardpen's change set, but the `update-version.sh` script should set this to `UNKNOWN`.  Can you manually change it to `UNKNOWN` for now and then as a follow up create an issue to handle it?

**Comment by richardpen** [comment]
👍

**Pull request merged**